components/_global-navigation.scss

```diff
@@ -143,9 +143,79 @@ $notification-counter-size: 18px;
 	}
 
 	&__search {
-		// TODO XW-1620
-		background-color: lightgray;
+		align-items: center;
+		border-top: $border-top-width solid $color-slate-gray;
+		display: flex;
 		flex: 2;
+		margin-top: -$border-top-width;
+		padding-left: 16px;
+
+		&__input-wrapper {
+			border-bottom: 1px solid $color-slate-gray;
+			display: flex;
+			flex: 1;
+		}
+
+		&__label {
+			align-items: center;
+			display: flex;
+			flex: 1;
+		}
+
+		&__input {
+			border: 0;
+			color: $color-dark-gray;
+			flex: 1;
+			font-size: $typescale-size-minus-1;
+			padding: 10px 4px 5px;
+
+			&:focus {
+				outline: none;
+			}
+
+			&::placeholder {
+				color: $color-dark-blue-gray;
+			}
+		}
+
+		&__close {
+			display: none;
+			padding: 3px;
+
+			&__icon {
+				color: $color-light-gray;
+				display: block;
+			}
+		}
+
+		&__submit {
+			display: none;
+			padding: 6px 9px;
+
+			&__icon {
+				display: block;
+				transform: rotate(180deg);
+			}
+		}
+
+		&.wds-is-active {
+			.wds-global-navigation__search {
+				&__input-wrapper {
+					margin-right: 15px;
+				}
+
+				&__input {
```

Comment:
As long as there are no more rules than the current ones it makes sense to keep it simple and without the nesting.